adding_species_manuscript.tex

```diff
@@ -500,7 +501,7 @@ \subsection*{Basic setup for chromosome-level simulations}
   At minimum, an average genome-wide recombination rate needs to be specified, which is typically available for well assembled genomes.
   For bacteria and archea, which primarily experience non-crossover recombination,
   the average tract length should also be specified
-  (see details in previous section).\\
+  (see details in previous section).
```

Comment:
The indentation was weird with a linebreak, I thought - inline in the paragraph is okay.